app/components/post_priority/post_priority_label.test.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import Tag from '@components/tag';
+import {PostPriorityType} from '@constants/post';
+import {renderWithIntlAndTheme} from '@test/intl-test-helper';
+
+import PostPriorityLabel from './post_priority_label';
+
+jest.mock('@components/tag', () => ({
+    __esModule: true,
+    default: jest.fn(),
+}));
+jest.mocked(Tag).mockImplementation((props) => React.createElement('Tag', {...props}));
+
+describe('PostPriorityLabel', () => {
+    it('should return null for standard priority', () => {
+        const {toJSON} = renderWithIntlAndTheme(
+            <PostPriorityLabel label={PostPriorityType.STANDARD}/>);
+        expect(toJSON()).toBeNull();
```

Comment:
Just curious, if you had brought this back into `main` branch, would this test pass?

I guess it would have a View with no icon and no label?